src/main/java/org/jabref/gui/entrytype/EntryTypeView.java

```diff
@@ -87,6 +88,10 @@ public EntryTypeView(LibraryTab libraryTab, DialogService dialogService, GuiPref
                   .load()
                   .setAsDialogPane(this);
 
+        Stage stage = (Stage) getDialogPane().getScene().getWindow();
```

Comment:
I have attempted a simple adjustment to the FXML (specifically setting the minWidth and minHeight properties in the root DialogPane), but it doesn't seem to affect the minimum window size, as the window itself can be resized to smaller than the set values. I believe this is because the FXML is encoding the internals and minimum size of those, while we need to set the minimum size for the stage as a whole. I have looked at incorporating the stage into the FXML, however, even if this path can work it seems to lead to a significantly more convoluted change to achieve the same effect. 

Please tell me if I have missed something, I am new to FXML and may just be missing something quite obvious.

Reply:
Since `EntryTypeView extends BaseDialog<EntryType>`, shouldn't that go into `org.jabref.gui.util.BaseDialog#BaseDialog`?

@Siedlerchr I see a big unclearness: Why don't the children of `BaseDialog` call the constructor? Or do I miss some JavaMagic (I would search for `super()` in the constructors).

Reply:
> If a constructor body does not begin with an explicit constructor invocation and the constructor being declared is not part of the primordial class Object, then the constructor body implicitly begins with a superclass constructor invocation "super();", an invocation of the constructor of its direct superclass that takes no arguments. 

https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.8.7